src/tests/ftest/launch.py

```diff
@@ -1442,33 +1448,46 @@ def install_debuginfos():
     # yum_base.processTransaction(rpmDisplay=yum.rpmtrans.NoOutputCallBack())
 
     # Now install a few pkgs that debuginfo-install wouldn't
-    cmd = ["sudo", "yum", "-y", "--enablerepo=*debug*", "install"]
+    cmd = ["sudo", "dnf", "debuginfo-install", "-y"]
     for pkg in install_pkgs:
         try:
             cmd.append(
                 "{}-{}-{}".format(pkg['name'], pkg['version'], pkg['release']))
         except KeyError:
             cmd.append(pkg['name'])
 
-    cmds.append(cmd)
+    cmd_list.append(cmd)
 
-    retry = False
-    for cmd in cmds:
-        try:
-            print(run_command(cmd))
-        except RuntimeError as error:
-            # got an error, so abort this list of commands and re-run
-            # it with a yum clean, makecache first
-            print(error)
-            retry = True
-            break
-    if retry:
-        print("Going to refresh caches and try again")
-        cmd_prefix = ["sudo", "yum", "--enablerepo=*debug*"]
-        cmds.insert(0, cmd_prefix + ["clean", "all"])
-        cmds.insert(1, cmd_prefix + ["makecache"])
-        for cmd in cmds:
-            print(run_command(cmd))
+    retry_count = 3
```

Comment:
Running this more than twice, with the `clean`/`makecache` between is not likely going to be any more successful than just running it twice.  The only reason two times might be more successful than one even is the `clean`/`makecache` between them.